### PR TITLE
Feature/show info message on new user

### PR DESCRIPTION
**Pull request by vinkabuki**

### CODE REVIEW

File: packages/desktop/src/renderer/components/widgets/channels/BasicMessage.tsx [hunk 1]
```diff
@@ -87,13 +85,9 @@ export const transformToLowercase = (string: string) => {
   const hasPM = string.search('PM')
   return hasPM !== -1 ? string.replace('PM', 'pm') : string.replace('AM', 'am')
 }
-
 export interface BasicMessageProps {
   messages: DisplayableMessage[]
   pendingMessages?: Dictionary<MessageSendingStatus>
-  // setActionsOpen: (open: boolean) => void
-  // actionsOpen: boolean
```
Comment (EmiM) [inline packages/desktop/src/renderer/components/widgets/channels/BasicMessage.tsx:95]: Leftovers

File: packages/state-manager/src/sagas/publicChannels/sendNewUserInfoMessage/sendNewUserInfoMessage.saga.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+import { MAIN_CHANNEL } from '../../../constants'
+
+export function* sendNewUserInfoMessageSaga(
+  action: PayloadAction<ReturnType<typeof publicChannelsActions.sendNewUserInfoMessage>['payload']>
+): Generator {
+  const community = yield* select(communitiesSelectors.currentCommunity)
+  const isOwner = community.CA
+
+  if (!isOwner) return
```
Comment (Dinek007) [inline packages/state-manager/src/sagas/publicChannels/sendNewUserInfoMessage/sendNewUserInfoMessage.saga.ts:24]: why no `if (!community.CA) return` ?

File: packages/state-manager/src/sagas/publicChannels/sendNewUserInfoMessage/sendNewUserInfoMessage.saga.test.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    ReturnType<typeof communitiesActions.addNewCommunity>['payload']
+    >('Community')
+    user = await factory.create<ReturnType<typeof identityActions.addNewIdentity>['payload']>(
+      'Identity',
+      {
+        id: community1.id
+      }
+    )
+
+    channel = (await factory.build<typeof publicChannelsActions.addChannel>('PublicChannel'))
```
Comment (siepra) [inline packages/state-manager/src/sagas/publicChannels/sendNewUserInfoMessage/sendNewUserInfoMessage.saga.test.ts:44]: Why do you need a channel here?

File: packages/state-manager/src/sagas/publicChannels/sendNewUserInfoMessage/sendNewUserInfoMessage.saga.ts (hunk 2 above)
Comment (Dinek007) [reply packages/state-manager/src/sagas/publicChannels/sendNewUserInfoMessage/sendNewUserInfoMessage.saga.ts:24]: maybe we should check it before action call and do not call this saga when its not an owner

**Review by siepra — APPROVED** [review]

**Review by Dinek007 — APPROVED** [review]

File: packages/state-manager/src/sagas/publicChannels/sendNewUserInfoMessage/sendNewUserInfoMessage.saga.ts (hunk 2 above)
Comment (vinkabuki) [reply packages/state-manager/src/sagas/publicChannels/sendNewUserInfoMessage/sendNewUserInfoMessage.saga.ts:24]: I'd leave all logic in sagas and keep eventChannel clean and have just one responsibility - dispatch actions

Comment (Dinek007) [reply packages/state-manager/src/sagas/publicChannels/sendNewUserInfoMessage/sendNewUserInfoMessage.saga.ts:24]: got it

**Pull request merged**